SCAutolib/models/gui.py

```diff
@@ -225,37 +258,81 @@ def wrapper(self, *args, **kwargs):
 class GUI:
     """Represents the GUI and allows controlling the system under test."""
 
-    def __init__(self, wait_time: float = 5):
+    def __init__(self, wait_time: float = 5, res_dir_name: str = None):
         """Initializes the GUI of system under test.
 
         :param wait_time: Time to wait after each action
+        :param custom_dir_name: Provide a custom name of the results dir under
+        /tmp/SC-tests/. The default is `timestamp`.
```

Comment:
with this change, the we need to update the documentation too to reflect the default naming